features/supplier/supplier_account.feature

```diff
@@ -7,23 +7,181 @@ Background:
+    | Company size              | Large                                         |
+    | VAT number                | 410802502                                     |
+
+  # Can pull the dunsNumber from the supplier and use it with this step
+  And I see 'that supplier.dunsNumber' text on the page
+
+  # This button should only appear once all details are filled in.
+  And I see the 'Save and confirm' button
+
+  When I click the summary table 'Change' link for 'Registered company name'
+  Then I am on the 'Registered company name' page
+  And I enter 'Toys "ᴙ" Not Us' in the 'Registered company name' field and click its associated 'Save and return' button
+
+  Then I click the summary table 'Change' link for 'Registered company address'
+  And I am on the 'What is your registered office address?' page
+  And I enter '101 Liquidation Street' in the 'Building and street' field
+  And I enter 'Shutdown' in the 'Town or city' field
+  And I enter 'N0 TOY' in the 'Postcode' field
+  And I enter 'Wales' in the 'location-autocomplete' field and click its associated 'Save and return' button
+
+  Then I click the summary table 'Change' link for 'Registration number'
+  And I am on the 'Are you registered with Companies House?' page
+  And I choose 'Yes' radio button
+  And I enter '18092232' in the 'companies_house_number' field and click its associated 'Save and return' button
+
+  Then I click the summary table 'Change' link for 'Trading status'
+  And I am on the 'What’s your trading status?' page
+  And I choose 'other' radio button
+  And I click 'Save and return'
+
+  Then I click the summary table 'Change' link for 'Company size'
+  And I am on the 'What size is your organisation?' page
+  And I choose 'Micro' radio button
+  And I click 'Save and return'
+
+  Then I click the summary table 'Change' link for 'VAT number'
+  And I am on the 'Are you registered for VAT?' page
+  And I choose 'No' radio button
+  And I click 'Save and return'
+
+  Then I click the summary table 'Change' link for 'DUNS number'
+  And I am on the 'Change your DUNS number' page
+  And I click the 'Return to company details' link
+
+  # Not checking DUNS number here, as the value is random
+  Then I see the 'Registration information' summary table filled with:
+    | field                     |  value                                                |
+    | Registered company name   | Toys "ᴙ" Not Us                                       |
+    | Registered company address| 101 Liquidation Street Shutdown N0 TOY United Kingdom |
+    | Registration number       | 18092232                                              |
+    | Trading status            | Other                                                 |
+    | Company size              | Micro                                                 |
+    | VAT number                | Not VAT registered                                    |
+
+  # Can pull the dunsNumber from the supplier and use it with this step
```

Comment:
Should we remove the comments, or at least some of them, before merging?

Reply:
I'm leaving them in for information. It's quite a long scenario and they make it easier to understand what's going on at certain stages, an to explain some little quirks, I think.

Reply:
ok 👍